presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreenState.kt

```diff
@@ -0,0 +1,8 @@
+package me.androidbox.presentation.event.screen
+
+import android.net.Uri
+import androidx.compose.runtime.mutableStateListOf
+
+data class EventScreenState(
```

Comment:
Whats the use of this state?

Reply:
@MKiperszmid this will handle of the events that are used by the eventScreen